Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1518,6 +1524,7 @@
     <RegEx find="dranj" replaceWith="deranj" />
     <RegEx find="dsek" replaceWith="dsjek" />
     <RegEx find="[dt]cj?eni" replaceWith="dcijeni" />
+    <RegEx find="cj?enim" replaceWith="cijenim" />
```

Comment:
Yes, assuming `tcjenim` ==> `dcijenim` is the correct conversion, line 1527 should be,
```xml
    <RegEx find="(?&lt;![dt])cj?enim" replaceWith="cijenim" />
```